spec/controllers/houses_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe HousesController, type: :controller do
+  include Devise::TestHelpers
+  
+  describe "GET index" do
+    let (:user) {create :user}
```

Comment:
these come from factory bot.  you'll need to add that gem, then create the factories for them.

https://github.com/thoughtbot/factory_bot_rails